src/LayerTreeNode/LayerTreeNode.spec.jsx

```diff
@@ -18,7 +18,8 @@ describe('<LayerTreeNode />', () => {
     const wrapper = TestUtil.mountComponent(LayerTreeNode, defaultProps, {
       context: {
         rcTree: {
-          prefixCls: ''
+          prefixCls: '',
+          registerTreeNode: () => {}
```

Comment:
I tried updating some days ago and failed to create a sane context back then. Happy this good now.